ssg/build_yaml.py

```diff
@@ -1032,6 +1042,16 @@ def __str__(self):
         return self.id_
 
 
+def rule_filter_from_def(filterdef):
+    if filterdef is None or filterdef == "":
+        return None
+    def filterfunc(rule):
+        # Remove globals for security and only expose
+        # variables relevant to the rule
+        return eval(filterdef, {}, rule.__dict__)
```

Comment:
Neat!

Reply:
```suggestion
        return eval(filterdef, {"__builtins__": None}, rule.__dict__)
```

According to https://stackoverflow.com/a/9558001/592892 this can be improved, but eval is still dangerous, although that modification restricts the attack surface.